README.md

```diff
@@ -1,6 +1,6 @@
 # Certified AWS Associate Developer Notes
 
-### 2022 AWS developer associate exam 
+### 2023 AWS developer associate exam 
```

Comment:
Can we use the exam version (DVA-C02) instead of using the year, year will require a yearly update.